cluster/kubemark/config-default.sh

```diff
@@ -31,9 +31,9 @@ MASTER_DISK_SIZE=${MASTER_DISK_SIZE:-20GB}
 REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-false}
 PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 
-MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-debian}
```

Comment:
Swtich kubemark in 1.4 branch to use GCI master. 

 @gmarek @wojtek-t for confirmation.